README.md

```diff
@@ -29,17 +29,17 @@ Go SDK for the [Auth0](https://auth0.com/) Management API.
 
 ### Requirements
 
-- Go 1.17+
+This library follows the [same support policy as Go](https://go.dev/doc/devel/release#policy). The last two major Go releases are actively supported and compatibility issues will be fixed. While you may find that older versions of Go may work, we will not actively test and fix compatibility issues with these versions.
```

Comment:
Not 100% convinced this extra wording is worth it

Reply:
Yes, agreed.